core-server/src/main/java/org/orbisgis/orbisserver/coreserver/web/MainController.java

```diff
@@ -136,7 +136,19 @@ public Result login() throws IOException {
     public Result processList(@Parameter("token") String token) throws IOException {
         for(Session session : sessionList) {
             if (session.getToken().toString().equals(token)) {
-                return ok(render(processList, "processList", session.getOperationList()));
+                List<Operation> opList = session.getOperationList();
+                List<Operation> importExportList = new ArrayList<Operation>();
+
+                for(Operation op : opList){
+                    for(String keyword :  op.getKeyWord()){
+                        int index = 0;
+                        if(keyword.equals("Export") || keyword.equals("Import")){
+                            importExportList.add(op);
```

Comment:
I just want the list without the keyword import and export. The lists for import and export will be created in the route data/import  and data/export. 

But i think yet to the shape of these list in the web page. if there are lot of element in each list, he must be good to put them in the nav left like the process.